daml-lf/transaction/src/main/protobuf/com/daml/lf/transaction.proto

```diff
@@ -49,26 +44,26 @@ message Node {
         NodeFetch fetch = 4;
         NodeExercise exercise = 5;
         NodeLookupByKey lookup_by_key = 6;
-        NodeRollback rollback = 7; // *since version 14*
+        NodeRollback rollback = 7;
     }
 
-    string version = 63; // *since 11*, optional
+    string version = 63;
 }
 
 message KeyWithMaintainers {
     oneof key {
-        com.daml.lf.value.VersionedValue key_versioned = 1; // *until version 11*
-        bytes key_unversioned = 3; // *defined in version >= 1.11*
+        com.daml.lf.value.VersionedValue key_versioned = 1; // *must not be used*
```

Comment:
Protobufs have a way to mark a field as deprecated: `[deprecated = true]` (see https://protobuf.dev/programming-guides/proto3/#options). Should we use this here?

Reply:
Let start with that in daml 3.0